go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/cosmos/cosmos-sdk/db v1.0.0-beta.1
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/iavl v0.19.6
-	github.com/cosmos/ledger-cosmos-go v0.13.2
+	github.com/cosmos/ledger-cosmos-go v0.12.4
```

Comment:
ledger-cosmos-go release notes are sparse but 0.12.4 and 0.13.2 should contain the hotfix needed for Ledger to work on MacOS. 

Ref: https://github.com/cosmos/ledger-cosmos-go/releases